picoquic/quicctx.c

```diff
@@ -1236,12 +1236,12 @@ int picoquic_connection_error(picoquic_cnx_t* cnx, uint32_t local_error)
         cnx->local_error = local_error;
         cnx->cnx_state = picoquic_state_disconnecting;
 
-        DBG_PRINTF("Protocol error (%x)", local_error);
+        DBG_PRINTF("Protocol error (0x%x)", local_error);
     } else if (cnx->cnx_state < picoquic_state_client_ready) {
         cnx->local_error = local_error;
         cnx->cnx_state = picoquic_state_handshake_failure;
 
-        DBG_PRINTF("Protocol error %x", local_error);
+        DBG_PRINTF("Protocol error 0x%x", local_error);
     }
 
     return PICOQUIC_ERROR_DETECTED;
```

Comment:
Good fixes, but unrelated. We should break them in a separate PR.

Reply:
Yep